packages/react-native-renderer/src/ReactFabricRenderer.js

```diff
@@ -232,7 +232,11 @@ const ReactFabricRenderer = ReactFiberReconciler({
     if (__DEV__) {
       const oldIsInAParentText = parentHostContext.isInAParentText;
```

Comment:
We're only going in one direction. If oldIsInAParentText is true, you can bail out early.

Reply:
That's fair.

Reply:
Isn't Text > View > Text valid while Text > View > (plain text) isn't?

Reply:
`Text` -> `View` is not valid.

(It _currently_ is for iOS but it soon won't be. Tim is changing this now.)

Reply:
cc @yungsters in case this is not correct

Reply:
Wait why? What do you do when you need to put a view inline in some text?

Reply:
Certain types of components (like `Image`) _are_ allowed inside of `Text` - but `RCTView` itself (the `<View>` component) is not.

Hm. But I think I see what you're saying. Only treating this value as a one-way thing would suggest that this is allowed: `<Text>` -> `<Image>` -> "foo"

Reply:
Maybe this is a better example of why we shouldn't only set this value one way:
```
<Text>
  <ScrollView>
    foo
  </ScrollView>
</Text>
```